test/persistent_receiver_test.go

```diff
@@ -148,6 +150,35 @@ var _ = Describe("PersistentReceiver", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(fmt.Sprint(receiver)).To(ContainSubstring(fmt.Sprintf("%p", receiver)))
 		})
+		Context("with a connected messaging service and specially formatted queue", func() {
+            slog.SetLogLoggerLevel(slog.LevelDebug)
+            const basicQueueName = "test_invalid_durability_on_bind_raises_exception"
+            modifiedQueueName := "#P2P/QTMP/v:solbroker/" + basicQueueName
+
+			BeforeEach(func() {
+                fmt.Println("Starting test")
```

Comment:
@TrentDaniel, I don't think we should have this print statement here.

Reply:
Fixed in https://github.com/SolaceDev/pubsubplus-go-client/commit/33a49baf262fe9ed0b7b41fd899ffe3e42f8ab56, please resolve this conversation if you agree.